test/test_tensor_creation_ops.py

```diff
@@ -790,6 +790,75 @@ def compare_helper_(like_fn, t):
                     tp = t.permute(p)
                     compare_helper_(like_fn, tp)
 
+    def _test_special_splits(self, torch_fn, np_fn, dim_range, dim, device, dtype):
```

Comment:
Also add tests in TestViewOps that validate these operations return views of the original tensor